packages/@n8n/task-runner/src/task-runner.ts

```diff
@@ -452,15 +452,15 @@ export abstract class TaskRunner extends EventEmitter {
 			});
 		});
 
-		this.send({
-			type: 'runner:rpc',
-			callId,
-			taskId,
-			name,
-			params,
-		});
-
 		try {
+			this.send({
+				type: 'runner:rpc',
+				callId,
+				taskId,
+				name,
+				params,
+			});
+
```

Comment:
There's no `catch` block here? Shouldn't we error the task if RPC fails?

Reply:
The RPC calls are initiated by user code. Hence it's up to the user code to handle or ignore them. If the user code throws, we of course error the task.